spec/javascript/fixtures/short_form_submit_payload.js

```diff
@@ -35,7 +35,7 @@ const payload = {
        phone: '9548883321',
        phoneType: 'Cell',
        maritalStatus: 'Married',
-       applicationId: 'a0n0x000000AbE6AAK' },
+       appMemberId: 'a0n0x000000AbE6AAK' },
```

Comment:
Salesforce doesn't seem to actually care what we send, but we were actually sending the appMemberId instead of the applicationId so I've renamed the field to make it clear. 

If we find later that we do in fact have to send the applicationId as part of the application member, we can add it in.